app/(dashboard)/profile/page.tsx

```diff
@@ -56,37 +50,7 @@ const ProfilePage = async () => {
             />
           </div>
         </div>
-        <div className='m-auto mt-10 w-5/6'>
-          <h1 className='m-5 text-lg font-medium md:pl-20 lg:pl-40'>
-            My donated items:
-          </h1>
-
-          {fetchedItems && fetchedItems.length > 0 ? (
-            <ul className='flex flex-col items-center gap-10'>
-              {fetchedItems.map((item) => (
-                <li key={item.id}>
-                  <ItemCard
-                    imageSrc={item.imageSrc}
-                    item_name={item.item_name}
-                    condition={item.condition}
-                    postcode={item.postcode}
-                    postable={item.postable}
-                    itemId={item.id}
-                  />
-                  <Modal
-                    name='Delete Item'
-                    itemId={item.id}
-                    message='By pressing Confirm you will delete this item'
-                  />
-                </li>
-              ))}
-            </ul>
-          ) : (
-            <h2 className='m-5 text-lg font-thin'>
-              You have not donated any items.
-            </h2>
-          )}
-        </div>
+        <DisplayDonatedItems userId={user.id}></DisplayDonatedItems>
```

Comment:
```suggestion
        <DisplayDonatedItems userId={user.id} />
```
I'm only being pedantic about self-closing components but this change of yours was sweet ✨